tests/test_workflows.py

```diff
@@ -1025,6 +1023,11 @@ def test_inline_images():
     with pytest.raises(KeyError) as exc:
         reader.pages[2]._get_image(("test",))
 
+    url = "https://github.com/py-pdf/pypdf/files/15233597/bug1065245.pdf"
+    name = "iss2598c.pdf"  # test coming from another test in test_image.py
```

Comment:
What should this comment tell us? Couldn't we make this more specific?

Reply:
The test file being used somewhere else I've prefered to add note. comment is updated